overdrive.py

```diff
@@ -854,6 +854,8 @@ def book_info_to_metadata(cls, book, include_bibliographic=True, include_formats
                         format_id, overdrive_id
                     )
 
+                # TODO: while reviewing the code, saw the medium variable
+                # is not used after declaration.  Consult with original coder and fix.
```

Comment:
Good catch. I think you can take this out. In book_info_to_metadata we use mediaType to get a value for medium.